src/NuGetPackageConfigConverter/Services/IConverterUpdateProvider.cs

```diff
@@ -7,6 +7,6 @@ namespace NuGetPackageConfigConverter
 {
     public interface IConverterViewProvider
     {
-        Task ShowAsync(Solution sln, Action<ConverterUpdateViewModel, CancellationToken> model);
+        Task  ShowAsync(Solution sln, Action<ConverterUpdateViewModel, CancellationToken> model);
```

Comment:
Nit: some of these files have extra white space

Reply:
yep